### PR TITLE
Added timer to connection

### PR DESCRIPTION
**Pull request by hAdamm**

### CODE REVIEW

File: src/Connection.php [hunk 1]
```diff
@@ -77,6 +106,30 @@ public function __construct($config, $name = null) {
...
+     */
+    public function getTotalTime() {
+        return $this->measureTotalTime ? $this->measuredTime : null;
+    }
+
+    /**
+     * @param string $timerName
+     * @return void
+     */
+    public function startTimer($timerName) {
```
Comment (ZZromanZZ) [inline src/Connection.php]: What about allow $timerName = null ?

File: src/Connection.php [hunk 2]
```diff
@@ -30,6 +30,31 @@ class Connection extends \Dibi\Connection {
...
 
     /**
+     * @var string
+     */
+    const MEASURE_TOTAL_TIME_KEY = "measureTotalTime";
+
+    /**
+     * @var bool
+     */
+    protected $measureTime = false;
```
Comment (ZZromanZZ) [inline src/Connection.php]: What about $debugTime or $queryTime, and and with postfix enabled, $queryTimeEnabled = false ?

File: tests/ConnectionTest.phpt [hunk 3]
```diff
@@ -82,6 +82,41 @@ class ConnectionTest extends \Tester\TestCase {
...
+        $this->connection->$methodName("SELECT * FROM [Contacts]");
+
+        $t2 = $this->connection->readTimer("t2");
+        Assert::same($this->connection->readTimer("t1"), $t2);
+
+        $this->connection->startTimer("t3");
+        $this->connection->$methodName("SELECT * FROM [Contacts]");
+        $t3 = $this->connection->readTimer("t3");
+
+        Assert::same($this->connection->readTimer("t1"), $t2 + $t3);
```
Comment (lordjim666) [inline tests/ConnectionTest.phpt]: no root namespace?

Comment (hAdamm) [inline tests/ConnectionTest.phpt]: no

**Pull request merged**